.github/workflows/browser-compatibility-test-previous-major-version.yml

```diff
@@ -0,0 +1,309 @@
+name: Browser Compatibility Test
+
+on:
+  schedule:
+    # More information on cron https://crontab.guru/
+    # GitHub actions is using UTC time. Scheduling action at 12 am PST
+    - cron: '0 8 * * *'
+
+env:
+  SELENIUM_GRID_PROVIDER: saucelabs
+  CLOUD_WATCH_METRIC: false
+  TEST_TYPE: Browser-Compatibility-Test
+  SAUCE_USERNAME: ${{secrets.SAUCE_USERNAME}}
+  SAUCE_ACCESS_KEY: ${{secrets.SAUCE_ACCESS_KEY}}
+  MESSAGING_USER_ARN: ${{secrets.MESSAGING_USER_ARN}}
+  SLACK_JS_SDK_DEV_CORE_WEBHOOK: ${{secrets.SLACK_JS_SDK_DEV_CORE_WEBHOOK}}
+
+jobs:
+  get-previous-version:
+    name: Get Previous Major Version
+    runs-on: ubuntu-latest
+    outputs:
+      previous_version_number: ${{ steps.get-version.outputs.version-number }}
+    steps:
+      - name: Checkout Package
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Get Version
+        id: get-version
+        run: |
+          current_version=$(.github/script/get-current-version)
+          echo "::set-output name=version-number::$(($current_version - 1))"
```

Comment:
Does this syntax work? I tried `echo "$((3.0.0-1))"` and it threw an arithmetic error for me.

Reply:
Ah I'll have to edit it then, I thought it just outputted the first number